src/main/java/org/generationcp/middleware/domain/study/StudyEntrySearchDto.java

```diff
@@ -0,0 +1,95 @@
+package org.generationcp.middleware.domain.study;
+
+import org.generationcp.middleware.domain.etl.MeasurementVariable;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@AutoProperty
+public class StudyEntrySearchDto {
+
+	private int studyId;
+
+	private List<MeasurementVariable> entryDescriptors;
```

Comment:
Why use`MeasurementVariable` class? It seems we are only using the name ****of the variable in the usage of this Search Dto

Reply:
We may be interested on using the alias too, so this was is easier to add any extra information